### PR TITLE
chore(rabbitmq): refactor assertEntity in tests

### PR DESCRIPTION
**Pull request by mmorel-35**
## What does this PR do?

Refactors assertions on entities in `rabbitmq_test.go`

This applies the assertions in each function and mutualizes duplicated code

## Related issues

Related to https://github.com/testcontainers/testcontainers-go/pull/2855#discussion_r1820404094

### CODE REVIEW

**Review by mdelapenya — DISMISSED** [review]
LGTM, thanks for your efficiency and quickness 🙇

File: testing.go [hunk 1]
```diff
@@ -147,3 +148,18 @@ func isCleanupSafe(err error) bool {
...
+
+// RequireContainerExec is a helper function that executes a command in a container
+// It insures that there is no error during the execution
+// Finally returns the output of its execution
+func RequireContainerExec(ctx context.Context, t *testing.T, container Container, cmd []string) string {
+	t.Helper()
+
+	code, out, err := container.Exec(ctx, cmd)
+	require.NoError(t, err)
+	require.Zero(t, code)
```
[the remark this answers is not in the document]
Comment (stevenh) [reply testing.go:160]: A follow up PR if it's needed I would suggestion.

[the remark this answers is not in the document]
Comment (mdelapenya) [reply testing.go:160]: As it's pretty well documented at the function comments, lgtm as is.

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Pull request merged**